plugins/liquid/auth.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+define('DOKU_AUTH', dirname(__FILE__));
+define('AUTH_USERFILE', DOKU_CONF.'users.auth.php');
+class auth_plugin_liquid extends DokuWiki_Auth_Plugin {
```

Comment:
Since you're here and building images, can you fix https://github.com/CRJI/EIC/issues/292 too pls?

Reply:
done, added `plugin.info.txt`